app/src/main/java/org/simple/clinic/contactpatient/views/CallPatientView_Old.kt

```diff
@@ -0,0 +1,104 @@
+package org.simple.clinic.contactpatient.views
```

Comment:
It seems like this is a new file rather than rename. Can you confirm this change, please?